libdnf5/base/transaction.cpp

```diff
@@ -1048,6 +1048,7 @@ bool Transaction::Impl::check_gpg_signatures() {
     libdnf5::rpm::RpmSignature rpm_signature(base);
     std::set<std::string> processed_repos{};
     int num_checks_skipped = 0;
+    std::set<std::string> repo_of_checks_skipped;
```

Comment:
I don't get this variable name. Wouldn't something like `repos_with_skipped_checks` be more intuitive?